### ISSUE TITLE
Flowchart? Diagram?

### ISSUE THREAD

**Issue by mattansb** [easystats/easystats#26]
Although the package names have high face validity, it still might not be 100% clear if after fitting a model I should use `parameters` or `estimate`? For a Bayesian model, should I *also* or *only* use `bayestestR`? Maybe a combination of these? And what exactly the difference is between some of the functionality between these packages?

I think it could be useful to have some sort of flowchart or diagram to help users navigate the various `easystats` packages (inspired by the [new `insight` figure](https://raw.githubusercontent.com/easystats/insight/master/paper/figure2.png)). A sort of "So you've fit a model - what do you want to do next?" kind of thing.

**Comment by DominiqueMakowski** [comment]
Yes, this could go in the website #25  and here on the README.

However, I believe the organisation will also become clearer once better documentation is there. For instance, I think that the main point of entry into the easyverse for total beginners will be higher level packages such as `report`. It will include in its vignette some very basic and introductory tutorials to R and modelling. From there, however, users will be able to access a more specialised documentation contained in more specialised packages ("click here to learn more about standardized parameters" -> parameters' vignettes). At the same time, these more specialised packages will also be the point of entry for an audience looking for precise stuff ("how to get ICC for mixed models in R" -> performance; "bayes factors savage dickey" -> bayestestR). 

In the end, I expect (and hope) the hierarchical structure of easystats to make sense, smoothly conveying relevant information to the relevant audience.

But yes, a diagram explaining the theorethical and philosophical structure of easystats would be great :)

**Comment by strengejacke** [comment]
_bump_

**Comment by DominiqueMakowski** [comment]
Can't believe one year has already passed... 'twas so fast

**Comment by strengejacke** [comment]
Yes, time is running. But on the other hand, we have put 8 packages on CRAN, all beyond just a preliminary state.

But back on topic: I think what we could do is probably really think about some "workflow" vignettes, that demonstrate the nice interplay of packages. And about the caveats. Just because we have samples of simulated model parameters from `simulate_model()` and thus have a "posterior" distribution that can be used with many _bayestestR_ functions, it doesn't mean that our (frequentist) model uses the same sophisticated sampling algorithm like Bayesian models do, and thus inteference is limited, at least not the same as in Bayesian statistics.

**Comment by DominiqueMakowski** [comment]
Definitely! With also like seemingly basic introductory articles: "how to do a logistic regression analysis in R" and we could here show parameters, performance, see and modelbased for visualization, effectsize, and then bayestestR for the "and now you can do it all again super easily using the Bayesian framework" - it could also reach new users for which easystats is a good place to start with R and stats

**Comment by IndrajeetPatil** [comment]
I think we should follow a template like the following (cf. [here](https://github.com/mlr-org)):

![image](https://user-images.githubusercontent.com/11330453/158169527-1d42df8e-111b-4af2-b819-2c532ec680b2.png)
